@pauliescanlon/gatsby-mdx-embed/src/components/GeneralObserver/GeneralObserver.tsx

```diff
@@ -9,26 +9,28 @@ import React, {
 interface GeneralObserverProps {
   /** Fires when IntersectionObserver enters viewport */
   onEnter?: (id?: string) => void
+  height?: number
 }
 
 export const GeneralObserver: FunctionComponent<GeneralObserverProps> = ({
   children,
-  onEnter
+  onEnter,
+  height = 400
```

Comment:
I think we'll need to keep the default height as `0` for instance the [TwitterFollowButton](https://gatsby-mdx-embed.netlify.app/?path=/docs/components-twitter-twitterfollowbutton--page#twitterfollowbutton) is only about 50px. I think having a height prop is a good idea and we should probably do some work on the other components that also have a height prop and pass it through as you've done for CodePen but i think we'll still need a default height of `0`... what do you think?

Reply:
> i think we'll still need a default height of 0... what do you think?

Bit of remediation work to be done on this, maybe if it's put back to `0` but we make an Issue to add the `height` prop to the components that need to have a specified height??

Reply:
That makes a lot of sense to me. This was the more controversial aspect of my PR to me. The changes to the CodePen component itself are pretty straight forward, but with the GeneralObserver being used by so many components and coming down to personal preference, it was a bit touchier. 

I can put that default height value to `0` for now, I think that's a good call.